tests/unit/test_workflows.py

```diff
@@ -548,10 +571,41 @@ def test_to_dict(self, wf):
             },
             'id': None,
             'state': 'INITIAL',
+            'custom_payload':{},
         }
 
         wf.nodes['10'].to_dict.assert_called()
 
+
+        wf.custom_payload["foo"] = "bar"
+        res = wf.to_dict()
+        assert res == {
```

Comment:
This test is a kind of redundant. 
I'd like to keep the tests clean and practical.

```
    def test_to_dict_with_custom_payload(self, wf):
        wf.custom_payload["foo"] = "bar"
        res = wf.to_dict()
        assert res['custom_payload']['foo'] == 'bar'
```
This should be good enough.

Reply:
sure thing